src/Sylius/Bundle/CustomerBundle/composer.json

```diff
@@ -36,21 +36,18 @@
     ],
     "require": {
         "php": "^8.2",
-        "doctrine/orm": "^2.13",
-        "egulias/email-validator": "^3.1",
+        "doctrine/orm": "^2.18",
+        "egulias/email-validator": "^4.0",
         "sylius/customer": "^2.0",
         "sylius/resource-bundle": "^1.12",
         "symfony/framework-bundle": "^6.4.1 || ^7.1",
-        "webmozart/assert": "^1.9"
-    },
-    "conflict": {
-        "doctrine/orm": ">= 3.0.0"
```

Comment:
Why did you remove this conflict?

Reply:
Because in this bundle, we have in the main requirements section: "doctrine/orm": "^2.18", so this explicit requirement restricts the installation of a higher version than 2.x